commons/clever-cloud-overview/intro-to-apps.md

```diff
@@ -20,14 +20,17 @@ An application is defined on Clever Cloud by the following elements:
 
 If one of these elements is missing, Clever Cloud can't deploy your application properly (except the configuration file, optional in some cases).
 
-Available languages are listed below:
+Available languages and techno are listed below:
```

Comment:
Maybe change that to something like "Supported plaforms:"

Btw, could you rewrite the commit message in english please?